project_euler/problem_41/sol1.py

```diff
@@ -0,0 +1,54 @@
+from math import sqrt
+from typing import List
+from itertools import permutations
+
+"""
+We shall say that an n-digit number is pandigital if it makes use of all the digits
+1 to n exactly once. For example, 2143 is a 4-digit pandigital and is also prime.
+What is the largest n-digit pandigital prime that exists?
+"""
+
+"""
+All pandigital numbers except for 1, 4 ,7 pandigital numbers are divisible by 3.
+So we will check only 7 digit panddigital numbers to obtain the largest possible
+pandigital prime.
+"""
+
+
+def is_prime(n: int) -> bool:
+    """
+    Returns True if n is prime,
+    False otherwise.
+    >>> is_prime(67483)
+    False
+    >>> is_prime(563)
+    True
+    >>> is_prime(87)
+    False
+    """
+    if n % 2 == 0:
+        return False
+    for i in range(3, int(sqrt(n) + 1), 2):
+        if n % i == 0:
+            return False
+    return True
+
+
+def compute_pandigital_prime(n: int) -> List[int]:

```

Comment:
This function does not return one prime but instead returns a list of primes.
```suggestion
def compute_pandigital_primes(n: int) -> List[int]:

```